main.go

```diff
@@ -1,14 +1,77 @@
 package main
 
 import (
-	"ava-go/bot"
+	"ava-go/handlers"
+	"errors"
 	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/bwmarrin/discordgo"
 )
 
+func start() (err error) {
+	token, ok := os.LookupEnv("BOT_TOKEN")
+	if !ok {
+		return errors.New("couldn't find environment variable $BOT_TOKEN")
+	}
+
+	goBot, err := discordgo.New("Bot " + token)
+	if err != nil {
+		return errors.New("bot auth: " + err.Error())
+	}
+
+	// declare intents (needed to be able to get member info)
+	goBot.Identify.Intents = discordgo.MakeIntent(discordgo.IntentsAll)
+	goBot.AddHandler(handlers.MessageHandler)
+	goBot.AddHandler(handlers.VoiceStateHandler)
+
+	// add slash command handlers
+	goBot.AddHandler(func(s *discordgo.Session, i *discordgo.InteractionCreate) {
+		if h, ok := handlers.CommandHandlers[i.ApplicationCommandData().Name]; ok {
+			h(s, i)
+		}
+	})
+
+	if err = goBot.Open(); err != nil {
+		return errors.New("bot open: " + err.Error())
+	}
+
+	log.Println("bot is running!")
+
+	// add commands
+	registeredCommands := make([]*discordgo.ApplicationCommand, len(handlers.Commands))
+	for i, v := range handlers.Commands {
+		cmd, err := goBot.ApplicationCommandCreate(goBot.State.User.ID, "379276406326165515", v)
```

Comment:
we should probably move this ID to an environment variable or .env file at some point

Reply:
yeah, end goal is a mongodb to store this in addition to role info, but .env files would be a good idea

Reply:
imo mongodb is good for application state, but not as much for application configuration. tokens/client ids are generally configuration over state

Reply:
Maybe open an issue for this?